### PR TITLE
Add goreleaser

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - linux
+    goarch:
+      - amd64
+# Archive customization
+archive:
+  format: tar.gz
+  replacements:
+    amd64: 64-bit
+    darwin: macOS
+    linux: Tux
```
Comment (greut) [inline .goreleaser.yml]: no needs for those customization.

File: .goreleaser.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      - windows
+      - darwin
+      - linux
+    goarch:
+      - amd64
+
+sign:
+  cmd: gpg
+  args: ["-a", "-u", "ops@exoscale.ch", "--detach-sign", "${artifact}"]
+  artifacts: all
```
Comment (greut) [inline .goreleaser.yml]: for the time being, `artifacts: none`

Comment (pierre-emmanuelJ) [reply .goreleaser.yml]: okay and i have see your PR on goreleaser

**Comment by greut** [comment]
we can remove the `cmd-*` stages then.

**Comment by greut** [comment]
I'd put `cs` in the name

```
dist/egoscale_SNAPSHOT-054b882657462bc189f3a747fd22009ca918307f_linux_amd64.tar.gz
```

**Pull request merged**